Singularity/Singularity/Screen/ScreenClasses/GameScreen.cs

```diff
@@ -384,7 +377,14 @@ private void SettlerBuild(object sender, EventArgs eventArgs, Vector2 v, Settler
             // TODO eventually the EPlacementType should be instance but currently that
             // TODO requires a road to be place and therefore throws an exception !!!!!
 
-            var cCenter = PlatformFactory.Get(EPlatformType.Command, ref mDirector, v.X - 55, v.Y - 100);
+            // adds the command center to the GameScreen, as well as two general units
+            
+            var graphid = IdGenerator.NextiD();
+            mDirector.GetDistributionDirector.AddManager(graphid);
+            
+            var cCenter = PlatformFactory.Get(EPlatformType.Command, ref mDirector, v.X - 55, v.Y - 100, commandBlueprint: false);
+            // CommandCenter cCenter = new CommandCenter(new Vector2(v.X-55, v.Y-100), mCylPlat, mBlankPlat, ref mDirector, false);
+
             AddObject(cCenter);
 
             var genUnit = new GeneralUnit(cCenter, ref mDirector, cCenter.GetGraphIndex());
```

Comment:
Here we give the "real" Id to the genUnits of the command center, the one before won't even be connected to anything